src/django/api/facilities_download_view_set.py

```diff
@@ -42,6 +45,22 @@ def list(self, request):
         Returns a list of facilities in array format for a given query.
         (Maximum of 250 facilities per page.)
         """
+        facility_download_limit, created = FacilityDownloadLimit.objects.get_or_create(
+            user=request.user,
+            defaults={
+                "allowed_downloads": 10,
```

Comment:
I would suggest adding `FacilitiesDownloadSettings.DEFAULT_DOWNLOADS_LIMIT` and using it instead of hard coding the value.